src/AppBundle/Subscriber/AutoLabelPRFromContentSubscriber.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace AppBundle\Subscriber;
+
+use AppBundle\Event\GitHubEvent;
+use AppBundle\GitHubEvents;
+use AppBundle\Issues\GitHub\CachedLabelsApi;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Looks at new pull requests and auto-labels based on text.
+ */
+class AutoLabelPRFromContentSubscriber implements EventSubscriberInterface
+{
+    private $labelsApi;
+
+    public function __construct(CachedLabelsApi $labelsApi)
+    {
+        $this->labelsApi = $labelsApi;
+    }
+
+    /**
+     * Adds a "Needs Review" label to new PRs.
+     *
+     * @param GitHubEvent $event
+     */
+    public function onPullRequest(GitHubEvent $event)
+    {
+        $data = $event->getData();
+        if ('opened' !== $action = $data['action']) {
```

Comment:
It's just a matter of adding it later - it could be really easy, i actually didn't check :)